src/constructorio.js

```diff
@@ -60,6 +61,7 @@ class ConstructorIO {
       testCells,
       fetch,
       trackingSendDelay,
+      sendTrackingEvents: sendTrackingEvents === false ? false : true, /* eslint-disable-line no-unneeded-ternary */
```

Comment:
Defaulting to `true` unless the option is specifically false

Reply:
I think I'd prefer a line above that does

```
if (!sendTrackingEvents) {
    sendTrackingEvents = true;
}
```

above, so we don't have to do the weird unneeded ternary dance?